src/DelegationRegistry.sol

```diff
@@ -120,8 +123,26 @@ contract DelegationRegistry is IDelegationRegistry, ERC165 {
     /**
     * See {IDelegationRegistry-getDelegationsForAll}.
     */
-    function getDelegationsForAll(address vault) external view returns (address[] memory) {
-        return delegationsForAll[vault][vaultVersion[vault]].values();
+    function getDelegationsForAll(address vault) external view returns (address[] memory delegates) {
```

Comment:
I think this (as well as the other `getDelegations...` functions) should probably return `address[] memory delegates, uint256[] memory expiries`.

The reasoning is something along the lines of, "If I want to enumerate my active delegations, I probably want to also know when each active delegation expires"